static/css/shop/_main.scss

```
@@ -0,0 +1,66 @@
+.shop {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.cards {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  align-self: stretch;
+}
+
+.card {
+  max-width: 18em;
+  margin-top: 1em;
+}
+
+.card > .image {
+  position: relative;
+}
+
+.card > .image > img {
```

Try to use SCSS features for better readability.
```suggestion
.card {
  max-width: 18em;
  margin-top: 1em;
  >.image {
    position: relative;
    >img {
      width: 100%;
    }
  }
}
```
However, I wouldn't block the merge because of this. Just note for the future.